api/src/main/java/org/pmiops/workbench/billing/FreeTierBillingService.java

```diff
@@ -34,7 +35,7 @@
 public class FreeTierBillingService {
```

Comment:
Can we add documentation for this class and add comments for methods to

Reply:
what documentation would you find useful here?

Reply:
just basic javacdoc comments describing the purpose of the class